labs/lab-1/FillOneRegularPothole.k

```diff
@@ -0,0 +1,26 @@
+function fillpotholes () { 
+	repeat (5) { 
+		if (frontIsClear () ) { 
+			move () ; 
+			turnRight () ; 
+			move () ; 
+			putBeeper () ;
+		} 
+		if (frontIsBlocked () ) { 
```

Comment:
I think this is ALWAYS going to be true here and the `if` condition can be removed, simplifying the code.